### ISSUE TITLE
Issue with "Выражения и операторы": (short summary here please)

### ISSUE THREAD

**Issue by thinkjazz** [mdn/translated-content#916]
MDN URL: https://developer.mozilla.org/ru/docs/Web/JavaScript/Guide/Expressions_and_Operators

#### What information was incorrect, unhelpful, or incomplete?
Operators должны быть переведены как операции. Эта ошибка перевода кочует еще с Си. Мы можем обратиться к русскому переводу Флэнагана Д. JavaScript Полное руководство 7 издание. Так же можно воспользоваться для определения этого термина operators(eng) - операции(rus) в https://context.reverso.net/translation/english-russian/operators#%D0%BE%D0%BF%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D0%B8 
Как пример:
These operators return a cell range of zero, ore or more cells.
Эти операции возвращают диапазон из нуля, одной или более ячеек.
Или ещё
minus sign, see also operators
знак "минус", см. также операции
plus sign, see also operators
знак "плюс", см. также операции
division sign, see also operators
знак деления, см. также операции

Тут следует уточнить, что ++ \ -- инкремент и декремент могут быть как операциями так и операторами(если идут в в новой строке и отделяются ; ). Давайте уже покончим с этой неразберихой. 
#### Specific section or headline?
Выражения и операции

#### What did you expect to see?
Операции 

1. Выражения присваивания
2. Операции сравнения
3. Арифметические выражения
4. Бинарные операции
5. Логические выражения
6. Строковая операция сложения(конкатенация)
7. Условная (тернарная) операция
8. Операция запятая
9. Унарные операции
10. Выражения отношений
10.1 Операции равенства и неравенства
      10.1.1 Операция строгого равенства
      11.1.2 Операция строгого неравенства
 10.2 Операции сравнения
11. Приоритет операций

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `ru/web/javascript/guide/expressions_and_operators`
* MDN URL: https://developer.mozilla.org/ru/docs/Web/JavaScript/Guide/Expressions_and_Operators
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/ru/web/javascript/guide/expressions_and_operators/index.html
* Last commit: https://github.com/mdn/translated-content/commit/39fb0931c082da04dfb29b1043d14fd7bdafbcab
* Document last modified: 2021-05-17T15:45:20.000Z

</details>

**Comment by mpstv** [comment]
@thinkjazz спасибо за участие!

Правильно ли я понял, что предлагается `operator` переводить то как `операция`, то как `оператор` в зависимости от контекста?

**Comment by thinkjazz** [comment]
> @thinkjazz спасибо за участие!
> 
> Правильно ли я понял, что предлагается `operator` переводить то как `операция`, то как `оператор` в зависимости от контекста?

operators - стоит переводить как операция.  Но в контексте ++ и -- можно и как оператор и как оперпция

**Comment by mpstv** [comment]
Я думаю, что это хорошее замечание.

Почитал википедию про [операции](https://ru.wikipedia.org/wiki/%D0%9E%D0%BF%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8F_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)) и [операторы](https://ru.wikipedia.org/wiki/%D0%9E%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)) в программировании.

С одной стороны - формально действительно правильнее переводить `operator`-> `операция`, с другой - возможно, все уже привыкли к переводу `operator`-> `оператор`.
Мне кажется, что решение о том как переводить эти термины в документации mdn - это важное решение, которое нуждается дискусии.

@lex111 @captainspring @myshov @Saionaro @sashasushko @armanpwnz что думаете?

**Comment by thinkjazz** [comment]
> Конечно, хочется как правильно, но бывают ситуации, когда язык принял неверную форму и идти против неё нет никакого смысла.
> 
> Я предлагаю подправить какую-нибудь маленькую статью, где есть употребления operator и operators (вдруг ещё и operations :-) и в пулл реквесте обсудить её. Может, даже призвать внешних наблюдателей.

Я согласен с вами, и окончательно решение, конечно за русскоязычным сообществом.

**Comment by sashasushko** [comment]
Хочу возродить ишью, потому что по работе зашёл на https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

И задумался, а это оператор или операция?

**Comment by thinkjazz** [comment]
В переводе Луиса Атенсию Функциональное программирование на JavaScript от 2018 года, на 88 странице описывается сравнения императивной и декларативной программы, как раз там операция и в скобка операторов.

**Comment by thinkjazz** [comment]
> Хочу возродить ишью, потому что по работе зашёл на https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
> 
> И задумался, а это оператор или операция?

Выражение условного доступа, вроде как.

выражение ?. идентификатор
выражение ?. [ выражение ]

В JavaScript не располагают свойствами только два значения, `null `и `undefined`. В обыкновенном выражении доступа к свойствам, использующем. или `[ ],` вы получите ошибку ТуреЕггог, если результатом вычисления выражения  слева оказывается `null `или undefined. Чтобы защититься от ошибок такого типа, вы можете применять синтаксис ?. и ?. [ ].
Давайте обсудим выражение a?.b. Если а равно null или undefined, тогда выражение вычисляется как undefined без попытки доступа к свойству b. Если а имеет какое-то другое значение, то результатом вычисления` а ? .Ь `становится
то, что получилось бы при вычислении` а .b `(и если а не имеет свойства по имени Ь, тогда значением снова будет `undefined`). Показанную форму выражения доступа к свойствам иногда называют “необязательным
связыванием”, поскольку она также работает для более длинных “связанных” выражений доступа к свойствам, например:
```
let а = { b: null };
a.b?.c.d // => undefined

```

**Comment by myshov** [comment]
C этим переводом большая путаница в книгах и на википедии. "operator" нужно всегда переводить как "оператор", а "operation" как "операция". В новом переводе Фланагана была допущена ошибка в переводе, в шестом издании "operator" переводился как "оператор".

Возьмём например вот это [корректное предложение на английском языке](https://www.ibm.com/docs/en/itcam-app-mgr/7.2.0?topic=tesl-operators-expressions-1):

> Arithmetic operators perform mathematical operations such as addition and subtraction with operands

Логично его перевести как:

> Арифметические операторы выполняют математические операции такие как сложение и вычитание над операндами

То есть три сущности:

- operation — некое абстрактное действие
- operator — конкретный символ представляющий это действие
- operand — объект над, которым производится это действие

**Comment by thinkjazz** [comment]
> 
То есть можно выделить три сущности:

operation (операция) — некое абстрактное действие
operator (оператор) — конкретный символ, представляющий действие
operand (операнд) — объект, над которым производится действие

Хорошая идея
